cq-prolyfill.js

```diff
@@ -317,10 +318,26 @@ function preprocessSheet(sheet, callback) {
  *                                    success or empty string on failure
  */
 function loadExternal(href, callback) {
+	var cacheEntryType = typeof requestCache[href];
+	if (cacheEntryType === 'string') {
+		callback(requestCache[href]);
+		return;
+	}
+	else if (cacheEntryType === 'object') {
```

Comment:
Just double checking: will `requestCache[href]` ever be `null` (eg: failed xhr)?  If so, might be worth a truthiness check.